scripts/hydra

```diff
@@ -355,4 +347,18 @@ function status() {
   fi
 }
 
-eval "$(argc --argc-eval "$0" "$@")"
\ No newline at end of file
+# @cmd Remote install to your nodes the dependencies to run the metagraph using Ansible
```

Comment:
```suggestion
# @cmd Remotely configure cloud instances using Ansible
```

Is there a way to give extended help instructions so if you run `hydra remote_configure --help` it has more output? If so, it would be nice to mention the location of the ansible files that need to be edited.

Reply:
Sure, I'll add